.gitignore

```diff
@@ -16,3 +16,6 @@ vendor/
 bin/
 dist/
 .goreleaser.yml
+
+# Editor specific directories
+.vscode/
```

Comment:
Interesting: vscode is my main IDE and I haven't had to add the vscode hidden directory to any gitignore when working on git-enabled repositories. I poked at my global .gitignore as well and didn't see any .vscode items. Any idea on why this is necessarily? Mainly curious - this isn't blocking feedback.

Reply:
My .vscode directory was actually being tracked until I added this 🤷🏽‍♂️